.github/workflows/ios-maestro.yml

```
@@ -100,46 +115,225 @@ jobs:
                 with:
                     path: |
                         ~/Library/Caches/CocoaPods
                         ~/.cocoapods
-                    key: pods-global-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
+                    key: pods-${{ matrix.target }}-${{ runner.os }}-xcode-${{ env.XCODE_VERSION }}-${{ hashFiles('yarn.lock', format('{0}/ios/Podfile.lock', env.APP_DIR)) }}
                     restore-keys: |
-                        pods-global-${{ runner.os }}-
+                        pods-${{ matrix.target }}-${{ runner.os }}-xcode-${{ env.XCODE_VERSION }}-
+                        pods-${{ matrix.target }}-${{ runner.os }}-
+
+            -   name: Setup ccache
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)